rpcd/playbooks/rpc-pre-upgrades.yml

```diff
@@ -19,18 +19,63 @@
   roles:
     - { role: "rpc_pre_upgrade", tags: [ "rpc-pre-upgrades" ] }
 
-- name: restart beaver
+- name: remove beaver
   hosts: all
   tags:
     - rpc-pre-upgrades
+    - remove-beaver
   tasks:
     - name: Check if beaver is installed
       stat:
         path: /etc/init.d/beaver
       register: beaver_init_script
+      changed_when: false
```

Comment:
That would need to be changed in the original commit first. I presume it will also need it's own card.
For the backport, I think ti's fine, right?